### PR TITLE
Donnie pull request 2

### PR DESCRIPTION
**Pull request by Crateros**
Add css word-break. Still need to fix subtitle max width.

### CODE REVIEW

File: static/css/style.css [hunk 1]
```diff
@@ -23,14 +23,17 @@ body{
...
+}
+
 .subtitle-edit-container{
   height: 100vmin;
   overflow-y: scroll;
   overflow-x: hidden;
   padding-right: 6px;
 }
 
-
```
Comment (Tommy-Lim) [inline static/css/style.css:33]: delete this text

File: static/css/style.css [hunk 2]
```diff
@@ -23,14 +23,17 @@ body{
   padding-bottom: 3vmin;
 }
 
+.line1Box{
```
Comment (Tommy-Lim) [inline static/css/style.css]: add a comma then line2box if we want to apply these styles to both elements, or conversely, use only one class in the HTML

**Review by Tommy-Lim — COMMENTED** [review]
see comments for changes

File: static/css/style.css (hunk 2 above)
Comment (Crateros) [reply static/css/style.css]: I have changed this line to
`.line1Box, .line2Box{`

**Review by Tommy-Lim — APPROVED** [review]

**Pull request merged**